x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/columns.tsx

```diff
@@ -135,6 +138,10 @@ export const progressColumn = {
   'data-test-subj': 'mlAnalyticsTableColumnProgress',
 };
 
+export const getDFAnalyticsJobIdLink = (item: DataFrameAnalyticsListRow) => {
```

Comment:
Nit: this could be implicit return

Reply:
Changed this to implicit return in https://github.com/elastic/kibana/pull/65778/commits/f3c8367a68b4abffa1dff9f3b9b7c456ecc1552a#diff-5c7fbd428261fc2fd9567311e2e7046cL141.